hawc/templates/includes/epa/header.html

```diff
@@ -1,8 +1,8 @@
 <header class="masthead clearfix" role="banner">
     <img alt="The EPA logo" class="site-logo" src="https://www.epa.gov/sites/all/themes/epa/logo.png">
     <div class="site-name-and-slogan">
-        <h1 class="site-name"><a href="https://www.epa.gov/" rel="home" title="Go to the home page"><span>US
```

Comment:
reverted this changes; they're from the template:

- https://github.com/USEPA/OneEPA-Standalone-App-Template/blob/master/index.html#L43
- https://github.com/USEPA/OneEPA-Standalone-App-Template/blob/master/index.html#L63